frontend/src/components/ThemedButtons/OverlayButton.css

```diff
@@ -2,13 +2,14 @@
 	display: flex;
 	justify-content: center;
 	align-items: center;
+	align-self: center;
+	width: fit-content;
```

Comment:
Are you sure you want these? This button is always going to override align-items on the parent... Is it deliberate, so that you don't need to use align-items on every parent container that uses this button?